src/app/public/modules/text-editor/text-editor.component.scss

```diff
@@ -0,0 +1,54 @@
+@import "~@skyux/theme/scss/variables";
+@import "~@skyux/theme/scss/mixins";
+
+.sky-text-editor {
+
+  .editor-wrapper {
+    display: flex;
+    flex-wrap: wrap;
+    background-color: white;
+    width: 100%;
+    height: 300px;
+    padding: 1rem;
+    font-size: 1.2rem;
+    border: 1px solid #C0C0C0;
+    overflow-y: auto;
+    outline: none;
+  }
+
+  .sky-text-editor-toolbar {
+
+    .sky-toolbar-sectioned {
+      // !important is needed to override specificity from modern stylsheet in theme.
+      border-left: 1px solid $sky-border-color-neutral-medium !important;
+      border-right: 1px solid $sky-border-color-neutral-medium !important;
+    }
+
+    .sky-toolbar-item {
+      margin-right: 15px;
+    }
+
+    .sky-dropdown-button {
+      border: none;
+    }
+
+    sky-text-editor-menubar,
+    sky-text-editor-toolbar {
+      display: flex;
+      flex-wrap: wrap;
+    }
+
+  }
+}
+
+.sky-theme-modern {
+
+  .sky-text-editor {
+    .sky-text-editor-toolbar {
+      .sky-toolbar-sectioned {
+        border-top: 1px solid $sky-border-color-neutral-medium !important;
```

Comment:
is there a different name for this variable in modern? it is showing the default color (cdcfd2) instead of the modern color (d2d2d2)

Reply:
Yes. Updated to use the modern color.